packages/palette/src/elements/PhoneInput/PhoneInput.tsx

```diff
@@ -328,7 +332,7 @@ const caretMixin = css`
     border-left: 4px solid transparent;
     border-right: 4px solid transparent;
     border-top: 4px solid
-      ${({ disabled }) => {
+      ${({ disabled }: CaretProps) => {
```

Comment:
on line 312 `const caretMixing = css<CaretProps>` then here you don't need to declare the type